src/components/Upload/CommonFields/index.jsx

```diff
@@ -19,13 +19,11 @@
 import React from "react";
 import PropTypes from "prop-types";
 import IgnoreScm from "./IgnoreScm";
-import AccessLevel from "./AccessLevel";
 import OptionalAnalysis from "./OptionalAnalysis";
 import LicenseDecider from "./LicenseDecider";
 import UploadReuse from "./UploadReuse";
 
 function CommonFields({
-  accessLevel,
```

Comment:
I found the issue, on existing UI, the parameter is called as `accessLevel`

But the REST API reads it as header called `public`
https://github.com/fossology/fossology/blob/b1f0cf72f6dd6f16109b1edcf584c5feba0e6498/src/www/ui/api/documentation/openapi.yaml#L428-L438

Can you please make the changes accordingly to send the header as `public` and not `accessLevel`?